.github/workflows/test.yml

```diff
@@ -91,9 +91,6 @@ jobs:
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
         version: v2.5.0
-    - name: make upstream
```

Comment:
Not having this in a separate step mixes up the logs a bit. But hopefully it never fails now.